### PR TITLE
Making './scripts/cli.py microk8s setup' support controller config;

### PR DESCRIPTION
**Pull request by ycliuhw**
This change allows bootstrap with controller config:
```console
$ python3 ./scripts/cli.py microk8s setup --test-mode --config caas-image-repo=test-repo --config='logging-config="<root>=TRACE"'
```

### CODE REVIEW

File: scripts/cli.py [hunk 1]
```diff
@@ -525,7 +526,7 @@ def setup(controller, services, test_mode):
         fail_msg='Waited too long for addons to come up!',
     )
 
-    args = []
+    args = [i for c in [('--config', c) for c in config] for i in c]
```
Comment (knkski) [inline scripts/cli.py:529]: To make it a little easier to read, can we switch this to `args = [f'--config={c}' for c in config]`? (I believe Juju supports the `--config=foo=bar` syntax)

Comment (ycliuhw) [reply scripts/cli.py:529]: sure, this has much better readability.

**Comment by ycliuhw** [comment]
[setup Kubeflow CI test](https://trello.com/c/YsO7GfRB/1344-setup-kubeflow-ci-test)

**Review by knkski — APPROVED** [review]
Thanks for the PR :cake:

**Pull request merged**